Source/missiles.h

```diff
@@ -135,10 +128,7 @@ struct Missile {
 	int16_t lastCollisionTargetHash;
 };
 
-extern Missile Missiles[MAXMISSILES];
-extern int AvailableMissiles[MAXMISSILES];
-extern int ActiveMissiles[MAXMISSILES];
-extern int ActiveMissileCount;
+extern std::unordered_map<int32_t, Missile> Missiles;

```

Comment:
Should we maybe `reserve()` 125 to avoid rehashing during runtime for any normal range? @glebm might prefer it one way or the other depending on what hardware he targets :)

Reply:
Would be possible. 🙂

Reply:
I think with `std::list` this isn't needed anymore or?